### ISSUE TITLE
What is different between outlier and unlabelled points ?

### ISSUE THREAD

**Issue by hagianga21** [PRBonn/semantic-kitti-api#17]
Hi, What is different between outliers and unlabelled points?

**Comment by jbehley** [comment]
For the evaluation both are equally ignored. 

However, while labeling outlier correspond mainly to reflections or otherwise distorted measurement. 

In the dataset the main cause for outliers are windshields of cars that cause false measurements below the street. But I also so many cases with windows reflecting cars or buildings from the other side of the street.

**Comment by hagianga21** [comment]
Yeah, thanks for that. I am just a little curious. How about the reason for the unlabelled points? When I visualize the dataset. I also see unlabelled points class beside the outlier class.

**Comment by jbehley** [comment]
We labeled up to a range of 50 m each scan (but we used always the aggregated point clouds for labeling); sometimes it was also not possible within this range to definitively assign a class or it was unclear which class to assign. We then just left it unlabeled.

We choose 50 m, since it seemed reasonable at this maximal distance to decide which class is there with a single scan. But the decision is also debatable, of course.

Hope that explains your impression.
